### PR TITLE
VIH-7852 skip link accessability

### PR DESCRIPTION
**Pull request by FarronF**
### JIRA link (if applicable) ###
VIH-7852

### Change description ###
Moving navigation links in waiting room and participant login journey to the header to allow skip link to behave correctly. Also adding a return to hearings link to all waiting rooms as per Neil's request.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -70,47 +90,63 @@ describe('AppComponent', () => {
...
+                        useValue: activatedRouteMock
+                    }
+                ],
+                declarations: [
+                    AppComponent,
+                    TranslatePipeMock,
+                    MockComponent(BackNavigationComponent),
+                    MockComponent(HeaderComponent),
+                    MockComponent(FooterComponent),
+                    MockComponent(BetaBannerComponent)
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/app.component.spec.ts:125]: I've used this package a lot in previous projects. I find it helps hugely with UnitTests and keeping the tests focused on the specific component. I would recommend using this going forward.

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-space-routing.module.ts [hunk 2]
```diff
@@ -2,32 +2,34 @@ import { NgModule } from '@angular/core';
...
 import { ConferenceGuard } from '../security/conference.guard';
 import { ParticipantWaitingRoomGuard } from '../security/participant-waiting-room.guard';
+import { BackLinkDetails } from '../shared/models/back-link-details';
 import { pageUrls } from '../shared/page-url.constants';
 import { JohWaitingRoomComponent } from './joh-waiting-room/joh-waiting-room.component';
 import { JudgeWaitingRoomComponent } from './judge-waiting-room/judge-waiting-room.component';
 import { ParticipantWaitingRoomComponent } from './participant-waiting-room/participant-waiting-room.component';
 import { LoggedUserResolveService } from './services/logged-user-resolve.service';
 
+const returnText = 'judge-waiting-room.return-to-video-hearing-list';
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-space-routing.module.ts:12]: This translation path might change or be duplicated. Discussing with Lorenzo

**Review by omio-ma — APPROVED** [review]

**Review by mulvaneyjHMCTS — APPROVED** [review]

**Review by omio-ma — APPROVED** [review]

**Pull request merged**